core-feature-pack/common/src/main/resources/content/bin/common.bat

```diff
@@ -28,6 +28,12 @@ goto :eof
       set "DEFAULT_MODULAR_JVM_OPTIONS=!DEFAULT_MODULAR_JVM_OPTIONS! --add-exports=java.base/sun.nio.ch=ALL-UNNAMED"
       set "DEFAULT_MODULAR_JVM_OPTIONS=!DEFAULT_MODULAR_JVM_OPTIONS! --add-exports=jdk.unsupported/sun.misc=ALL-UNNAMED"
       set "DEFAULT_MODULAR_JVM_OPTIONS=!DEFAULT_MODULAR_JVM_OPTIONS! --add-exports=jdk.unsupported/sun.reflect=ALL-UNNAMED"
```

Comment:
According to my recently finished analysis all three --add-exports are not necessary:
 * --add-exports=java.base/sun.nio.ch=ALL-UNNAMED
 * --add-exports=jdk.unsupported/sun.misc=ALL-UNNAMED
 * --add-exports=jdk.unsupported/sun.reflect=ALL-UNNAMED
 
We can eliminate them.